lib/backend/k8s/k8s.go

```diff
@@ -471,6 +471,7 @@ func (c *KubeClient) listProfiles(l model.ProfileListOptions) ([]*model.KVPair,
 	if l.Name != "" {
 		kvp, err := c.getProfile(model.ProfileKey{Name: l.Name})
 		if err != nil {
+			log.Debugf("Error retrieving profile: %s", err)
```

Comment:
I think we generally prefer to use `log.WithError(err).Debug(...)` for this sort of thing.